fileuploads/templates/fileuploads/about.html

```diff
@@ -6,12 +6,32 @@
     <h1>About signalserver</h1>
     <div class="row">
         <div class="col-md-6">
-          <p>
-            signalserver is a web application that significantly reduces the amount of time required to perform quality control on large sets of video files. Built upon QCTools and FFmpeg, signalserver allows users to set their own Pass/Fail parameters, cross-referencing QCTools reports in an effort to identify potentially problematic outliers. Like MediaConch, but focused on frame-based video analysis rather than technical metadata policy checking, signalserver reduces human error by providing comprehensive and customizable collection-level quality control reporting.
-          </p>
-          <p>
-            signalserver is free and open source, and would not exist without the National Endowment for the Humanities, the Knight Foundation, and the FFmpeg community.
-          </p>
+             signalserver is a web application that significantly reduces the amount of time required to perform quality control on large sets of video files. Built upon QCTools and FFmpeg, signalserver allows users to set their own Pass/Fail parameters, cross-referencing QCTools reports in an effort to identify potentially problematic outliers. Like MediaConch, but focused on frame-based video analysis rather than technical metadata policy checking, signalserver reduces human error by providing comprehensive and customizable collection-level quality control reporting.
+
+             QCTools, MediaConch, and MediaArea, and now signalserver are all the products of a growing community of open-source developers dedicated to serving the needs of audiovisual preservationists. signalserver specifically addresses those who are engaged in large-scale workflows that require an automated means of quality controlling the mass amounts of digital files entering their repositories. Free to use and download and open to collaborators via the site's Github page, signalserver is an accessible solution to expensive proprietary software applications.
+
+             Bay Area Video Coalition receieved the funding for the development by the National Endowment of the Humanties and their Division of Preservation and Access over a two-year cycle (2015-2017). Additional funding was provided by the Knight Foundation. 
+
+             <h2>Contributors (2015-2017)</h2>
+
+             Dave Rice, QCTools Developer
+             Yayoi Ukai, Web Server Developer
+             Ashley Blewer, QCTools/SignalServer Developer
```

Comment:
We have a QCTools Developer, we have a signalserver Developer, and we have a combination QCTools/signalserver developer!